databricks/koalas/plot/plotly.py

```diff
@@ -171,3 +174,38 @@ def plot_box(data: Union["ks.DataFrame", "ks.Series"], **kwargs):
     fig["layout"]["xaxis"]["title"] = colname
     fig["layout"]["yaxis"]["title"] = "value"
     return fig
+
+
+def plot_kde(data: Union["ks.DataFrame", "ks.Series"], **kwargs):
+    from plotly import express
+    import databricks.koalas as ks
+
+    if isinstance(data, ks.DataFrame) and "color" not in kwargs:
+        kwargs["color"] = "names"
+
+    kdf = KdePlotBase.prepare_kde_data(data)
+    sdf = kdf._internal.spark_frame
+    sdf = sdf.select(*list(map(kdf._internal.spark_column_for, kdf._internal.column_labels)))
```

Comment:
`kdf._internal.data_spark_columns` instead of `*list(map(kdf._internal.spark_column_for, kdf._internal.column_labels))`?